ts/src/renderer/phaser/classes/PhaserUnit.ts

```diff
@@ -124,6 +140,16 @@ class PhaserUnit extends PhaserAnimatedEntity {
 		attrs: AttributeData[]
 	}): void {
 		console.log('PhaserUnit render-attributes', data); // TODO remove
+		if (!this.attributesContainer) {
+			// creating attributeContainer on the fly,
+			// only for units that have attribute bars
+			const sprite = this.sprite;
+			this.attributesContainer = this.scene.add.container(
```

Comment:
@sloont should we set attributes container origin to top center `(0.5, 0)` and label origin to bottom center `(0.5, 1)`?

Reply:
chat also probably bottom center `(0.5, 1)`

Reply:
phaser documentation sais we cant change container origin 
![image](https://user-images.githubusercontent.com/62058278/179207544-1e51e354-db39-4a94-94ef-00f0185710f1.png)